extra/man/alacritty.5.scd

```diff
@@ -9,6 +9,20 @@ Alacritty - TOML configuration file format.
 Alacritty's configuration file uses the TOML format. The format's specification
 can be found at _https://toml.io/en/v1.0.0_.
 
+# LOCATION
+
+Alacritty doesn't create the config file for you, but it looks for one in the
+following locations on UNIX systems:
+
+. `$XDG_CONFIG_HOME/alacritty/alacritty.toml`
+. `$XDG_CONFIG_HOME/alacritty.toml`
+. `$HOME/.config/alacritty/alacritty.toml`
+. `$HOME/.alacritty.toml`
+
+On Windows, the config file should be located at:
```

Comment:
We should reuse the "looks for"/search verbs here. There shouldn't be a config file there, it's all optional.